website/content/ja/contribute/projects/translations/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 翻訳
+list_pages: false
+---
+
+## 導入
+
+CNCFの素晴らしさの一端は関わっている人々の多様性とコミュニティからサポートされているからです。このコミュニティをより包括的にする一例として、可能な箇所でコンテンツの翻訳を提供することです。翻訳には当然それを作成およびメンテナンスする母語話者を必要とし、このページはTAG App Deliveryのコンテンツでのやり方を示します。
+
+## ガイドライン
+
+各翻訳で異なりますが、一般的なガイドラインやノウハウは[過去の翻訳PRs](https://github.com/cncf/tag-app-delivery/issues?q=label%3Atranslation+)や[Kubernetes SIG-Docs 翻訳ドキュメント](https://kubernetes.io/docs/contribute/localization)を参考にすることをおすすめします。
```

Comment:
```suggestion
各翻訳で異なりますが、一般的なガイドラインやノウハウは[過去の翻訳PR](https://github.com/cncf/tag-app-delivery/issues?q=label%3Atranslation+)や[Kubernetes SIG-Docsローカライゼーションドキュメント](https://kubernetes.io/ja/docs/contribute/localization/)を参考にすることをおすすめします。
```

IMO: 

1. It’s better to use the singular form
2. Translation -> Localization
3. A link to the Japanese documentation is better.